fabric-common/lib/impl/CryptoKeyStore.js

```diff
@@ -6,75 +6,100 @@
 */
 
 'use strict';
-const {Utils: utils} = require('../../');
+const {Utils} = require('../../');
 const jsrsasign = require('jsrsasign');
 const KEYUTIL = jsrsasign.KEYUTIL;
 
 const ECDSAKey = require('./ecdsa/key.js');
+const KeyValueStore = require('../KeyValueStore');
 
-/*
- * The mixin enforces the special indexing mechanism with private and public
+const _getKeyIndex = (ski, isPrivateKey) => {
```

Comment:
I know this is unchanged from the previous code but I can't help feeling it would be more readable (particularly from the calling side) as two functions:
```javascript
const _getPrivateKeyIndex = (ski) => {
    return ski + '-priv';
}

const _getPublicKeyIndex = (ski) => {
    return ski + '-pub';
}
```

Reply:
Indeed, I will add it in next PR as step 2.